projects/pipelines/testing/command_args.yml.j2

```
@@ -60,6 +60,7 @@ driver/cluster set_project_annotation/test_toleration:
   project: {{ base_image.namespace }}
 
 pipelines run_kfp_notebook:
+  user_count: {{ tests.pipelines.user_count }}
```

here you change that to:
```suggestion
  capture_extra_artifacts: {% if  tests.pipelines.user_count > 50 %}true{% else %}false{% endif %}
```
or you go one step ahead and define `tests.pipelines.extra_artifacts_threshold: 50`
this way, you can easily define `tests.pipelines.extra_artifacts_threshold: 5000` if you want to force the artifacts capture, or `tests.pipelines.extra_artifacts_threshold: 0` if you want to disable it

overall, best is to move all the constants to the config file ... so that they're not hard-coded constants and can be overwritten. (not an absolute rule though, it would be too heavy-weight to really pass everything this way 🙃 )